Controllers/AccountController.cs

```diff
@@ -8,10 +8,20 @@ namespace Chefster.Controllers;
 
 public class AccountController : Controller
 {
+    private static string GetRedirectUri()
+    {
+        if (Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") != "Development")
+        {
+            return "https://chefster.net/callback";
+        }
+        return "http://chefster.net/callback";
+    }
+
     public async Task LogIn()
     {
         var authenticationProperties = new LoginAuthenticationPropertiesBuilder()
             .WithRedirectUri(Url.Action("Profile", "Index")!)
+            .WithParameter("redirect_uri", GetRedirectUri())
```

Comment:
why do we need to add this? we specify the redirect uri on the line above right?